docs/faq.md

```diff
@@ -158,3 +158,10 @@ fallback profiles, or if you use specify a profile in the workspace.
 
 Copying the profile to `~/Library/MobileDevice/Provisioning Profiles` will
 resolve the error.
+
+## Why I do not see any Simulators after generating project using rules_xcodeproj?
+
+This can happen if you have opened Xcode using `rosetta`, the solution is to get native arm simulator support for your app and disable rosetta.
+
+![Alt text](/docs/xcode-rosetta.png?raw=true "Xcode without Rosetta")
```

Comment:
I'd prefer we not include the 0.5 MB image just for the FAQ as long as that's okay with you. I think the text description is enough since it's not necessarily just as simple as disabling the checkbox, right?

Reply:
The way we’ve handled images elsewhere is to upload them to GitHub, and use that link instead.

Reply:
(Just in case you still want to use the image.)

Reply:
Makes sense 👍 , removed the image.